src/index.ts

```diff
@@ -36,34 +35,22 @@ export async function run(options: RunOptions) {
   sourceMapSupport.install({
     environment: 'node',
   });
-
-  const cliArgs = parseArgs();
```

Comment:
Not sure how this got introduced, but this is wrong. Our CLI already parses and sends the parsed options to our run API.